### ISSUE TITLE
Add text mentioning that a link opens in a new tab when a link opens in a new tab

### ISSUE THREAD

**Issue by jbalboni** [department-of-veterans-affairs/va.gov-team#20401]
It appears that we should be providing guidance to screen reader users when links open in a new tab. We have lots of these links in VAOS.

We need to:
1. Figure out a consistent suffix to append to links in an `aria-label` attribute
2. Update all our new tab links to have the link text plus the suffix about opening in a new tab in an `aria-label` attribute

Resource: https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/links/links-and-new-windows/

**Comment by jbalboni** [comment]
@1Copenut @noahgelman We have a PR started up for this at https://github.com/department-of-veterans-affairs/vets-website/pull/16246, but have a couple questions

1. Should we show any visual indication (like an icon, or additional text) to non-screen reader users when a link opens in a new tab?
2. Is there any issue with using `aria-describedby` for the text indicating it's opening in a new tab? I wouldn't think so, but thought I'd ask.

cc @vbahinwillit

**Comment by noahgelman** [comment]
@jbalboni

1. An icon is not required for styling an external link, but it would be very good
2. `aria-label` would be the correct attribute to use. `aria-describedby` has a different functionality. `aria-describedby` allowed another element with an id to describe the element. That's not what the goal here is. The goal he is to add additional info to the current link text that informs the user about opening in a new tab

**Comment by laurenernest** [comment]
Let's proceed without the link icon for this ticket until we get broader guidance on the pattern.

**Comment by vbahinwillit** [comment]
@outerpress - What is your expert opinion on this approach

- https://medium.com/@svinkle/why-let-someone-know-when-a-link-opens-a-new-window-8699d20ed3b1

This is what was implemented. Thanks

**Comment by outerpress** [comment]
@vbahinwillit - I tested this out on the [demo](https://codepen.io/svinkle/pen/BreKRJ) from that article and it seems like a nice, elegant solution. Worked well with a screenreader and visually.

Just a couple of things to note/think about as its being implemented:
* Make sure it's necessary given the context. To pull an example from that article, it'd odd to show the icon on social media site icons (not a problem we have, I'm guessing). The pattern makes sense for the link it's applied to.
* The icon should always mean the same thing - in this case, you're going to an external (non-va) site, so we're opening that in a new window. If there are situations where we open a new window to an internal (VA.gov) site for some reason then we should make sure the VO text makes sense.

Also cc: @CrystabelReiter - in case this is an interesting pattern for the design system, or you've thought about something like this already ^^

**Comment by vbahinwillit** [comment]
Thanks for reading!

**Comment by vbahinwillit** [comment]
@noahgelman - What is your expert opinion on this approach for this issue?

https://medium.com/@svinkle/why-let-someone-know-when-a-link-opens-a-new-window-8699d20ed3b1

This is what was implemented. Thanks

**Comment by noahgelman** [comment]
@vbahinwillit @laurenernest @1Copenut 

I've tested the method suggested in the medium.com article as well discussed the details of implementing it with Trevor.

We really like their approach and approve of it's use.

It will require an update to a globally insert the hidden text so it's available everywhere.
